src/components/login/index.test.js

```diff
@@ -66,4 +69,18 @@ describe('LoginHOC', () => {
     expect(props.account).to.be.equal(account);
     expect(typeof props.activePeerSet).to.be.equal('function');
   });
+
+  it('should bind activeAccountSaved action to Login props.activeAccountSaved', () => {
+    const actionsSpy = sinon.spy(savedAccounts, 'activeAccountSaved');
+    wrapper.find('Login').props().activeAccountSaved();
+    expect(actionsSpy).to.be.calledWith();
+    actionsSpy.restore();
+  });
+
+  it('should bind dialogDisplayed action to Login props.dialogDisplayed', () => {
```

Comment:
`setActiveDialog` is no longer used in login component, so it can be removed and this test too.